app/Notifications/BatteryLow.php

```
@@ -0,0 +1,66 @@
+        return ['mail', WebhookChannel::class];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     */
+    public function toMail(object $notifiable): MailMessage
+    {
+        return (new MailMessage)->markdown('mail.battery-low', ['device' => $this->device]);
+    }
+
+    public function toWebhook(object $notifiable)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (undefined)

(UnusedFormalParameter)